src/components/chatbox/message.tsx

```diff
@@ -131,46 +132,55 @@ const AssistantMessage = ({
 					})}
 				>
 					<span className={s.divider} />
-					<div className={s.actionButtons}>
-						<Button
-							size="small"
-							color="secondary"
-							icon={
-								isCopied ? (
-									<IconCheckSquare24 height={12} width={12} />
-								) : (
-									<IconClipboard24 height={12} width={12} />
-								)
-							}
-							aria-label="Copy to clipboard"
-							text={isCopied ? 'Copied' : 'Copy'}
-							onClick={handleCopy}
-						/>
+					{
```

Comment:
This new conditional introduces the Thank you message
<img width="846" alt="CleanShot 2023-10-03 at 17 57 36@2x" src="https://github.com/hashicorp/dev-portal/assets/26389321/fe383d62-91a0-4e81-97e5-b67dbe193358">